build/Targets/Settings.props

```diff
@@ -71,6 +71,9 @@
 
     <RoslynNetSdkRootPath>$(MSBuildSDKsPath)\Microsoft.NET.Sdk\Sdk\</RoslynNetSdkRootPath> 
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
+
+    <ProjectAssetsFile Condition="'$(BuildingForLiveUnitTesting)' != 'true'">$(BaseIntermediateOutputPath)project.assets.json</ProjectAssetsFile>
+    <ProjectAssetsFile Condition="'$(BuildingForLiveUnitTesting)' == 'true'">$(LiveUnitTestingOriginalBaseIntermediateOutputPath)project.assets.json</ProjectAssetsFile>
```

Comment:
You don't define the special variables (BuildingForLiveUnitTesting), we define them, so that builds can be customized for LUT if your projects are customized in a way that we didn't anticipate or don't support.  It's an "escape hatch".

That said, I thought I knew why the *.nuget.g.props file wasn't being imported (thus requiring this special case), but it turns out that I don't.  I'll need to figure out where the Import of the nuget.g.props file actually is...

In